tests/test_converter.py

```diff
@@ -297,14 +324,14 @@ def test_does_not_rewrite_existing_frames_or_bounds_data(existing_dataset):
     # Frames + bbox data should not be modified
     assert os.path.getmtime(existing_dataset / "frame_0.png") == frame_0_time
     assert os.path.getmtime(existing_dataset / "frame_1.png") == frame_1_time
-    assert os.path.getmtime(existing_dataset / "bounds.json") == bounds_time
+    # assert os.path.getmtime(existing_dataset / "bounds.json") == bounds_time
 
     # Reference to frames and bounds data should still be present in the manifest
     manifest = {}
     with open(existing_dataset / "manifest.json", "r") as f:
         manifest = json.load(f)
-        assert manifest["bounds"] == "bounds.json"
         assert manifest["frames"] == ["frame_0.png", "frame_1.png"]
+        # assert manifest["bounds"] == "bounds.json"
```

Comment:
Removing tests for `bounds`? Or should they be uncommented?

Reply:
I'm leaving them commented out for now until bounds writing behavior is restored (if ever). I temporarily deleted it but there's an issue (#79) that would fix it.